var/spack/repos/builtin/packages/r-rpostgresql/package.py

```diff
@@ -27,3 +27,4 @@ class RRpostgresql(RPackage):
     depends_on('r@2.9.0:', type=('build', 'run'))
     depends_on('r-dbi@0.3:', type=('build', 'run'))
     depends_on('postgresql')
+    patch('config.guess.patch')
```

Comment:
I think we may be able to use the following and remove the patch. Can you try if this:
```suggestion
    
    depends_on('automake', type='build')
    
    patch_config_files = True

    @run_before('install')
    def patch_config_guess(self):
        AutotoolsPackage._do_patch_config_files(self)

```
works for you? I currently don't have access to any `aarch64` machine to verify it.

Reply:
I tried your fix on aarch64 and it completed the installation without errors.